### ISSUE TITLE
Fix AbstractPortsExtractorTest on Windows

### ISSUE THREAD

**Issue by apupier** [fabric8io/fabric8-maven-plugin#1203]
### Info

* f-m-p version : master
* If it's a bug, how to reproduce :
launch mvn clean install on master on Windows

```
Running io.fabric8.maven.generator.api.support.AbstractPortsExtractorTest
Tests run: 6, Failures: 0, Errors: 4, Skipped: 0, Time elapsed: 0.437 sec <<< FAILURE! - in io.fabric8.maven.generator.api.support.AbstractPortsExtractorTest
testKeyPatterns(io.fabric8.maven.generator.api.support.AbstractPortsExtractorTest)  Time elapsed: 0.298 sec  <<< ERROR!
java.nio.file.InvalidPathException: Illegal char <:> at index 2: /C:/git/fabric8-maven-plugin/generator/api/target/test-classes/io/fabric8/maven/generator/api/support/AbstractPortsExtractorTest-pattern-keys.yml
	at io.fabric8.maven.generator.api.support.AbstractPortsExtractorTest.extractFromFile(AbstractPortsExtractorTest.java:119)
	at io.fabric8.maven.generator.api.support.AbstractPortsExtractorTest.testKeyPatterns(AbstractPortsExtractorTest.java:43)

testConfigFileDoesNotExist(io.fabric8.maven.generator.api.support.AbstractPortsExtractorTest)  Time elapsed: 0.037 sec  <<< ERROR!
mockit.internal.MissingInvocation: 
Missing 1 invocation to:
io.fabric8.maven.core.util.PrefixedLogger#warn(String, Object[])
   with arguments: any String, "/bla/blub/lalala/config.yml"
   on mock instance: io.fabric8.maven.core.util.PrefixedLogger@75f9eccc
	at io.fabric8.maven.generator.api.support.AbstractPortsExtractorTest.testConfigFileDoesNotExist(AbstractPortsExtractorTest.java:92)

testReadConfigFromFile(io.fabric8.maven.generator.api.support.AbstractPortsExtractorTest)  Time elapsed: 0.03 sec  <<< ERROR!
java.nio.file.InvalidPathException: Illegal char <:> at index 2: /C:/git/fabric8-maven-plugin/generator/api/target/test-classes/io/fabric8/maven/generator/api/support/AbstractPortsExtractorTest.json
	at io.fabric8.maven.generator.api.support.AbstractPortsExtractorTest.extractFromFile(AbstractPortsExtractorTest.java:119)
	at io.fabric8.maven.generator.api.support.AbstractPortsExtractorTest.testReadConfigFromFile(AbstractPortsExtractorTest.java:35)

testAddPortToList(io.fabric8.maven.generator.api.support.AbstractPortsExtractorTest)  Time elapsed: 0.031 sec  <<< ERROR!
java.nio.file.InvalidPathException: Illegal char <:> at index 2: /C:/git/fabric8-maven-plugin/generator/api/target/test-classes/io/fabric8/maven/generator/api/support/AbstractPortsExtractorTest-pattern-values.yml
	at io.fabric8.maven.generator.api.support.AbstractPortsExtractorTest.extractFromFile(AbstractPortsExtractorTest.java:119)
	at io.fabric8.maven.generator.api.support.AbstractPortsExtractorTest.testAddPortToList(AbstractPortsExtractorTest.java:62)

Running io.fabric8.maven.generator.api.support.BaseGeneratorTest
false false
false false
true false
true false
false true
false true
true true
Tests run: 11, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.697 sec - in io.fabric8.maven.generator.api.support.BaseGeneratorTest

Results :

Tests in error: 
  AbstractPortsExtractorTest.testAddPortToList:62->extractFromFile:119 » InvalidPath
  AbstractPortsExtractorTest.testConfigFileDoesNotExist » MissingInvocation Miss...
  AbstractPortsExtractorTest.testKeyPatterns:43->extractFromFile:119 » InvalidPath
  AbstractPortsExtractorTest.testReadConfigFromFile:35->extractFromFile:119 » InvalidPath
```

**Comment by samolisov** [comment]
I've fixed this error in #1139

**Comment by samolisov** [comment]
The PR #1139 has been merged, so the issue can be closed.
